python/python/Ice/Future.py

```diff
@@ -57,6 +86,10 @@ def forwardCompletion(sourceFuture, targetFuture):
 
 
 class Future(FutureBase):
+    """
+    A Future object representing the result of an asynchronous operation.
```

Comment:
What is the relationship with https://docs.python.org/3/library/concurrent.futures.html ?

Reply:
We basically copied the API, so the code is compatible, then we have some additions in InvocationFuture.

I think we didn't use it directly because concurrent Futures have to be created by an executor. For example you create them using an executor.

It is not clear to me that this was the only possibility, for example couldn't IcePy implement its own executor and return concurrent.futures.Future? I don't think this is something we can rework for 3.8.

Reply:
I meant, should we document something like:
 - this API is similar / identical / compatible with concurrent Futures ?
 - you can convert a Future into a concurrent Future (or vice versa) using xxx

Reply:
I would improve in a follow-u PR.